test/models/test_ontology_submission.rb

```diff
@@ -8,7 +8,7 @@ def test_valid_ontology
 
     acronym = "BRO-TST"
     name = "SNOMED-CT TEST"
-    ontologyFile = "./test/data/ontology_files/BRO_v3.2.owl"
+    ontologyFile = "./test/data/ontology_files/BRO_v3.2.owl.gz"
```

Comment:
I don't agree with this change at all. It's not a good practice to modify existing unit tests (that were developed for a reason) to test new functionality. Pull request #144 that introduced support for gzip files should have come with it's own set of unit tests. There was even a request for unit tests in the pull request conversation, but it doesn't appear that they were ever added.

Reply:
thanks for the feedback,  overloading unit tests is less than ideal.

one of the purposes of this PR draft is to show that we are missing important unit tests.